OOP_HW1/src/test/java/ut_ticketsystem.java

```diff
@@ -0,0 +1,227 @@
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.catchThrowable;
+
+import java.util.Vector;
+
+public class ut_ticketsystem {
+    @Test
+    public void test_check_ticket_price_should_have_correct_price() {
+
+        Order ticket = new Order(TicketType.ADULT, 1);
+        int price = TicketSystem.CheckTicketPrice(ticket);
+
+        assertThat(price).matches(q -> q == 350);
```

Comment:
Maybe we can follow [AAA principle](https://thinkster.io/tutorials/blogs/the-aaa-structure-of-tests) on your test function. It will be more clarify what you test in your test function.
Here's the suggestion for you:
```suggestion
        Order ticket = new Order(TicketType.ADULT, 1);
        
        int price = TicketSystem.CheckTicketPrice(ticket);

        assertThat(price).matches(q -> q == 350);
```